anaconda_build/README.md

```diff
@@ -73,3 +73,58 @@ local development. E.g. on uframe-3-test, mi-instrument is installed under the r
 ```bash
 $ conda install mi-instrument
 ```
+
+## The stream/ingest engines and environment
+
+The recipes are located in the <i>enginemeta</i>, <i>ingestEngine</i> and <i>streamEngine</i> folders. The enginemeta recipe declares the dependencies used while installing the packages built from the other recipes (ingestEngine and streamEngine). Modify it only as those dependencies change. The other recipes are based on the whole codebases via clones of the following Git repositories.
```

Comment:
for consistency please name the directories so they are the same as the resulting package.  streamEngine should be stream_engine, ingestEngine should be ingest_engine

Reply:
The names now match